prow/jobs/kyma/components/application-proxy/application-proxy.yaml

```diff
@@ -51,5 +51,16 @@ presubmits: # runs on PRs
     labels:
       <<: *job_labels_template
       preset-build-release: "true"
+  - name: pre-rel08-kyma-components-application-proxy
```

Comment:
As in the comment below: `Component no longer exists, do not create jobs for new releases`

Reply:
Sorry, totally missed that comment, the job has been removed

Reply:
it has been removed, i totally missed that comment